huaweicloud/services/vpc/resource_huaweicloud_vpc_traffic_mirror_filter.go

```diff
@@ -0,0 +1,196 @@
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: VPC POST /v3/{project_id}/vpc/traffic-mirror-filters
+// API: VPC GET /v3/{project_id}/vpc/traffic-mirror-filters/{traffic_mirror_filter_id}
+// API: VPC PUT /v3/{project_id}/vpc/traffic-mirror-filters/{traffic_mirror_filter_id}
+// API: VPC DELETE /v3/{project_id}/vpc/traffic-mirror-filters/{traffic_mirror_filter_id}
+func ResourceVpcTrafficMirrorFilter() *schema.Resource {
```

Comment:
```suggestion
func ResourceTrafficMirrorFilter() *schema.Resource {
```
All functions shouldn't contains package name.

Reply:
fixed